### PR TITLE
add kruise charts v0.3.0

### PR DESCRIPTION
**Pull request by hellolijj**
Signed-off-by: Junjun Li <junjunli666@gmail.com>

### Ⅰ. Describe what this PR does

add kruise charts v0.2.0

### Ⅱ. Does this pull request fix one issue?

kruise charts is needed for latest release version

### CODE REVIEW

**Comment by FillZpp** [comment]
@hellolijj Please remove `apps_v1alpha1_cloneset.yaml` from the templates. We should only provide those [released CRDs](https://github.com/openkruise/kruise#install-crds).

File: charts/v0.3.0/templates/rbac_role.yaml [hunk 1]
```diff
@@ -0,0 +1,239 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (Fei-Guo) [inline charts/v0.3.0/templates/rbac_role.yaml]: Please reload this file.  https://github.com/openkruise/kruise/pull/193 updated this file.

Comment (hellolijj) [reply charts/v0.3.0/templates/rbac_role.yaml]: done

**Comment by hellolijj** [comment]
> @hellolijj Please remove `apps_v1alpha1_cloneset.yaml` from the templates. We should only provide those [released CRDs](https://github.com/openkruise/kruise#install-crds).

done

**Pull request merged**